pylint_odoo/test_repo/broken_module2/tests/data/odoo_data_noupdate_1.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+    </data>
+    <data>
+    </data>
+</odoo>
```

Comment:
AFAICS this case [does not pass pylint checks](https://travis-ci.org/OCA/pylint-odoo/jobs/240797733#L809) but it should.